base/message/example/self-issued-id-token.json

```diff
@@ -0,0 +1,9 @@
+{
+    "sub": "did:example:123456789abcdefghi",
+    "iss": "https://example.com",
+    "iat": 1610839200,
+    "exp": 1610842800,
+    "jti": "a4a711f6-35d6-49a1-9dd4-0b805b0c6e5b",
+    "bearer_access_scope": "scope1 scope2",
```

Comment:
```suggestion
```
This property (as defined [here](https://arnoweiss.github.io/decentralized-claims-protocol/#get-sts-token)) should not be part of this body but is instead part of the OAuth2 [token server request](https://arnoweiss.github.io/decentralized-claims-protocol/static/sts-openapi.yaml). That structure most def is a relevant payload but since it's url-form encoded, I'm not sure it makes sense to introduce it here. It should be disregarded for now.

Reply:
done